index.html

```diff
@@ -2812,6 +2812,32 @@ <h2>A project for first-time contributions</h2>
           </div>
         </div>
         <!-- ________ *Rabblerag* Contributor card END ________  -->
+		
+		  <!-- ________ *DATCYX* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Dionisio Datiles III</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="#">None</a>

```

Comment:
Replace username with your twitter username. if you don't have twitter replace twitter in this line and the line before with github and, replace the username with your github username. 
```suggestion
            <a href="https://www.twitter.com/username" target="_blank">@username</a>

```